ci/pipelines/pull-request/jinja.template.yml

```diff
@@ -158,7 +161,7 @@ jobs:
             status: pending
       - do:
         - put: concourse-metadata-resource
-        - task: create_instance
+        - task: create_instance-{{java_test_version.name}}
```

Comment:
Why are we changing the task names (create_instance, rsync_xyz, archive_results, etc), if we aren't doing jdk8 and jdk11 as separate tasks in the same job?

Reply:
this is not absolutely necessary as long as 8 and 11 jobs are split, so the main reason is just for consistency with the develop pipeline, to make it more explicit as you're looking at results which results you're looking at, and for future-proofing if 8 and 11 jobs ever get combined back together in the future.